src/pages/SpreadTrend/index.js

```diff
@@ -0,0 +1,146 @@
+import * as c3 from "c3";
```

Comment:
Suggest naming this SpreadTrendChart (consistent with making it clear what it's trying to draw)